packages/@angular/cli/lib/cli/index.ts

```diff
@@ -41,8 +41,9 @@ function loadCommands() {
 }
 
 export default async function(options: any) {
-  // ensure the environemnt variable for dynamic paths
-  process.env.PWD = path.normalize(process.env.PWD || process.cwd());
+  // Ensure the environment variables for dynamic paths are set.
+  // We can't use the existing process.env.PWD because it can be incorrect in shells like cygwin.
+  process.env.PWD = process.cwd();
```

Comment:
This is only used here: https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/models/schematic-command.ts#L100
If that is changed to use `process.cwd()`, the line here can be removed.  The following line appears completely unused as well.

Reply:
Done.